### PR TITLE
It's Time

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Yeah, these shouldn't be around anymore. They should have never been around to begin with, to be frank.

:cl: Fox McCloud
del: removes ponies
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/pony.dm [hunk 1]
```diff
@@ -1,129 +0,0 @@
...
-	status_flags = CANPUSH
-	universal_speak = 1
-
-/mob/living/simple_animal/pony/Life()
-	..()
-	if(stat == 2)
-		new /obj/item/reagent_containers/food/snacks/ectoplasm(src.loc)
-		src.visible_message("<span class='warning'>[src] lets out a contented sigh as [p_their()] form unwinds.</span>")
-		src.ghostize()
-		qdel(src)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/simple_animal/pony.dm]: What in the ever loving heck is this crap.

File: code/modules/mob/living/simple_animal/pony.dm [hunk 2]
```diff
@@ -1,129 +0,0 @@
...
-		return
-
-/mob/living/simple_animal/pony/attackby(var/obj/item/O as obj, var/mob/user as mob)  //Marker -Agouri
-	if(istype(O, /obj/item/soulstone))
-		O.transfer_soul("SHADE", src, user)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/simple_animal/pony.dm]: nice copy-pasta ya go goin' there.

File: code/modules/mob/living/simple_animal/pony.dm [hunk 3]
```diff
@@ -1,129 +0,0 @@
...
-	melee_damage_upper = 15
-	attacktext = "sends rainbows of power to"
-	minbodytemp = 0
-	maxbodytemp = 4000
-	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
-	speed = 0
-	stop_automated_movement = 0
-	status_flags = 0
-	faction = list("pony")
-	status_flags = CANPUSH
```
Comment (Fox-McCloud) [inline code/modules/mob/living/simple_animal/pony.dm]: I hear you like no status flags so we removed your status flags so we could set your status flags to `CANPUSH`

:ultrathonkong:

**Comment by KasparoVy** [comment]
Neighhhh

**Comment by Fox-McCloud** [comment]
😱

**Comment by Aurorablade** [comment]
I will do anything you want me to do just please no

**Comment by Kyep** [comment]
I support this.

**Comment by lass9030** [comment]
Tis a glorious day.

**Comment by Dinarzad** [comment]
RIP technicolor dwarf horses.

**Comment by Twinmold93** [comment]
I cannot think of a single time we have used them for anything practical, and they don't fit the general aesthetic. Most of the time, I'd say we forget they even exist. Time to remove 'em. Obviously any admin that would want to keep the sprites have the capability of grabbing them before they disappear.

File: code/modules/mob/living/simple_animal/pony.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/modules/mob/living/simple_animal/pony.dm]: Even Satan fears this

**Comment by AffectedArc07** [comment]
I support this

**Comment by Ralta** [comment]
There's been a lot of horsing around in the code - it's about time someone ponied up and fixed this. Just make sure you saddle up and hand the reigns over to someone else to check your work; we don't want any mistakes missed because galloped ahead to merge this. 

:horse: :horse_racing: :carousel_horse:

**Comment by Spacemanspark** [comment]
About time tbh

**Comment by Xsingular** [comment]
Please remove the Pokemon next, no reason they should stay either while we're at it. All the arguments that apply to the Ponies apply to the Pokemon as well (never used practically, don't fit the aesthetic etc etc...)

**Comment by Fox-McCloud** [comment]
They're not in the game or default sprites. If they were, I'd remove them too.

That's admins being admemes.

**Comment by improvedname** [comment]
Tfw my pr was the same and got closed.
Good riddance fox doing gods work

**Comment by Spacemanspark** [comment]
@Xsingular :P

**Comment by KasparoVy** [comment]
:carousel_horse::gun::cowboy_hat_face::ok_hand:

**Pull request merged**